pkgs/by-name/cu/curl-impersonate/firefox/curl-impersonate-0.6.1-apply-brotli-cmake.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Makefile.in b/Makefile.in
+index daed8c3..9a45245 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -178,6 +178,7 @@ brotli-$(BROTLI_VERSION).tar.gz:
+ 
+ $(brotli_static_libs): brotli-$(BROTLI_VERSION).tar.gz
+ 	tar xf brotli-$(BROTLI_VERSION).tar.gz
++	patch -d brotli-$(BROTLI_VERSION) -p1 < $(BROTLI_CMAKE_PATCH)
```

Comment:
Uh oh - `curl-impersonate-ff` uses ancient brotli which is at least vulnerable to CVE-2020-8927.
Considering curl-impersonate is working with internet traffic, this is *bad*. We should make an attempt to unvendor brotli instead if at all possible.

(github lists a bunch of .NET stuff for that CVE, but https://www.cve.org/CVERecord?id=CVE-2020-8927 says brotli prior to 1.0.8 is affected)

Reply:
After bringing this up in [the security discussions](https://matrix.to/#/!ATURSDtpSAgOTfvtbq:lossy.network/$gexP9cIUykokz7ef-mfeLo4hTPKXB2PW2sC3I_Gdhik?via=nixos.org&via=matrix.org&via=stratum0.org) matrix room, people also noticed `curl-impersonate-ff` uses old versions of boringssl (not ancient, but old) and does not reliably stay on top of curl updates and more. So yeah, security of this seems like a mess, i won't merge this. I appreciate your attempt at fixing the build, but looking at the security of this, a drop is the more appropriate direction to take this.

Reply:
I think this is fine actually, BROTLI_VERSION is 1.0.9 which is the release that fixed CVE-2020-8927. Not arguing with the broader point of -ff being out of date though.